src/sagemaker/workflow/pipeline.py

```diff
@@ -45,14 +45,14 @@ class Pipeline(Entity):
 
     Attributes:
         name (str): The name of the pipeline.
-        parameters (Sequence[Parameters]): The list of the parameters.
+        parameters (Sequence[Parameter]): The list of the parameters.
         pipeline_experiment_config (Optional[PipelineExperimentConfig]): If set,
             the workflow will attempt to create an experiment and trial before
             executing the steps. Creation will be skipped if an experiment or a trial with
             the same name already exists. By default, pipeline name is used as
             experiment name and execution id is used as the trial name.
             If set to None, no experiment or trial will be created automatically.
-        steps (Sequence[Steps]): The list of the non-conditional steps associated with the pipeline.
+        steps (Sequence[Step]): The list of the non-conditional steps associated with the pipeline.
```

Comment:
this doesn't match the code below: `steps: Sequence[Union[Step, StepCollection]] = attr.ib(factory=list)`